tests-ci/tests/test_dirty_rectangles.js

```diff
@@ -0,0 +1,990 @@
+px.import({scene:"px:scene.1.js",
+           assert:"../test-run/assert.js",
+           shots:"../test-run/tools_screenshot.js",
+           manual:"../test-run/tools_manualTests.js"}).then( function ready(imports) {
+
+var scene = imports.scene;
+var root = scene.root;
+var assert = imports.assert.assert;
+var shots = imports.shots;
+var manual = imports.manual;
+var isDirtyRectsEnabled = scene.dirtyRectanglesEnabled;
+var doScreenshot = shots.getScreenshotEnabledValue();
+var testPlatform=scene.info.build.os;
+
+var manualTest = manual.getManualTestValue();
+var timeoutForScreenshot = 40;
+
+var basePackageUri = px.getPackageBaseFilePath();
+
+// Use fontUrl to load from web
+var fontUrlStart = "https://sparkui.org/examples/fonts/";
+var IndieFlower = "IndieFlower.ttf";
+var fontIndieFlower = scene.create({t:"fontResource",url:fontUrlStart+IndieFlower});
+root.w=800;
+
+var bg = scene.create({t:"object", parent:root, x:100, y:100, w:1000, h:1000, clip:false});
+//var rect = scene.create({t:"rect", parent:root, x:100, y:100, w:700, h:700, fillColor:0x00000000, lineColor:0xFF00FF77, lineWidth:1, clip:false});
+var container = scene.create({t:"object", parent:root, x:100, y:100, w:800, h:600, clip:false});
+
+// Widgets for displaying metrics values 
+var height = scene.create({t:"text", parent:root, x:50, y:0, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"Height="});
+var width = scene.create({t:"text", parent:root, x:50, y:20, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"Width="});
+var boundsX1 = scene.create({t:"text", parent:root, x:200, y:0, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"BoundsX1="});
+var boundsY1 = scene.create({t:"text", parent:root, x:200, y:20, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"BoundsY1="});
+var boundsX2 = scene.create({t:"text", parent:root, x:200, y:40, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"BoundsX2="});
+var boundsY2 = scene.create({t:"text", parent:root, x:200, y:60, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"BoundsY2="});
+var drboundsX1 = scene.create({t:"text", parent:root, x:400, y:0, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"DrBoundsX1="});
+var drboundsY1 = scene.create({t:"text", parent:root, x:400, y:20, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"DrBoundsY1="});
+var drboundsX2 = scene.create({t:"text", parent:root, x:400, y:40, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"DrBoundsX2="});
+var drboundsY2 = scene.create({t:"text", parent:root, x:400, y:60, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"DrBoundsY2="});
+var testText = scene.create({t:"text", parent:root, x:600, y:60, textColor:0xFFDDFFFF, pixelSize:15,clip:false,text:"Test="});
+
+
+var url = "http://www.pxscene.org/examples/px-reference/gallery/images/apng/cube.png";
```

Comment:
Please change this url to be http://www.sparkui.org/examples/gallery/images/apng/cube.png

Reply:
Thanks Connie, I have done this change.